### PR TITLE
Feature/improve gutemberg detection

### PR DESCRIPTION
**Pull request by Landaluce**

### CODE REVIEW

**Comment by chantisnake** [comment]
just to mark this is the pr for issue: https://github.com/WheatonCS/Lexos/issues/639

File: lexos/managers/lexos_file.py [hunk 1]
```diff
@@ -128,7 +128,8 @@ def check_for_gutenberg(file_contents: str) -> bool:
         :return: a boolean representing if file is from Project Gutenberg.
         """
 
-        if re.search('Project Gutenberg', file_contents):
+        if re.search("\*\*\* START OF THIS PROJECT GUTENBERG.*?\*\*\*",
+                     file_contents):
```
Comment (mleblanc321) [inline lexos/managers/lexos_file.py]: should the message be stored in a constant?

Comment (mleblanc321) [reply lexos/managers/lexos_file.py]: kind of a hack, not sure how prevalent this "START OF ..." message is @gutenberg ?

Comment (chantisnake) [reply lexos/managers/lexos_file.py]: I agree this is a hack. We should use XML.

Comment (scottkleinman) [reply lexos/managers/lexos_file.py]: @chantisnake, I'm not sure how we would use XML for this. Can you explain?

Comment (chantisnake) [reply lexos/managers/lexos_file.py]: Hum, I am guessing that they have a specific XML tag at a specific place?

I can be wrong.

Comment (scottkleinman) [reply lexos/managers/lexos_file.py]: Sadly, they do not. The text is either plain text or HTML, but the HTML is not even that helpful.

Comment (chantisnake) [reply lexos/managers/lexos_file.py]: too bad, I need more investigation to give a definite answer to this problem

Comment (Landaluce) [reply lexos/managers/lexos_file.py]: they don't have any tags, and this is the only part of the Gutenberg boilerplates that is the same for all the Gutenberg files

**Comment by chantisnake** [comment]
I think this is an okay temporary solution. I think still think this is Hacky.

I personally think we need at least add a box on front-end to toggle whether to remove the boiler-plate.

**Comment by scottkleinman** [comment]
The pop-up warning _is_ very obtrusive. Perhaps something like this would provide a better user experience:

image


If anyone wants to do this, I got the image above by injecting the following code in the browser console right before the row containing the `mainscruboptions` div:

```html
<div class="row">
    <div class="col-md-10">
    <label style="color:red">Your documents appear to contain Project Gutenberg boilerplate.</label>
    <br>
    <label id="tmp" for="tmpbox" class="icon-checkbox checked">    
      <input type="checkbox" name="tmpbox" id="tmpnbox" checked="">Attempt to Remove Boilerplate<i class="fa fa-question-circle lexos-tooltip-trigger" data-toggle="tooltip" data-html="true" data-placement="right" data-container="body" title="Project Gutenberg boilerplate is annoying." style="padding-left:4px;font-size:18px;"></i>
     </label>
    </div>
</div>
```

**Pull request merged**